changelogs/unreleased/bump-metrics-to-3370f85.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua
+
+* Added a `tnt_election_leader_idle` metric to built-in `metrics`.
```

Comment:
Technically speaking, we can't add features into a bugfix release.

See [my comment](https://github.com/tarantool/tarantool/pull/9933#discussion_r1559996584) for the `release/3.0` backport.

Reply:
See updated changelog